packages/jest-runner/fuzz.ts

```diff
@@ -79,10 +78,13 @@ export const runInRegressionMode = (
 			});
 			return;
 		}
+		// Execute the fuzz test with each input file as no libFuzzer is required.
+		// Custom hooks are already registered via the jest-runner.
 		inputsPaths.forEach(([seed, path]) => {
-			const runOptions = fuzzerOptions.concat(path);
-			g.test(seed, () => {
-				return core.startFuzzingAsyncNoInit(fn, runOptions);
+			g.test(seed, async () => {
+				const content = await fs.promises.readFile(path);
```

Comment:
Is this preferable to `fs.readFileSync`?

Reply:
`readFileSync` blocks the event loop, so I would hesitate using it in the production part of the codebase.